source/includes/common/common-configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,370 @@
+           -d "grant_type=password" \
+           -d "client_id=CLIENT" \
+           -d "client_secret=SECRET"
+
+   - Replace the ``USER`` and ``PASSWORD`` with the credentials of a Keycloak user on the ``REALM``.
+   - Replace the ``CLIENT`` and ``SECRET`` with the client ID and secret for the MinIO-specific Keycloak client on the ``REALM``
+
+   You can process the results using ``jq`` or a similar JSON-formatting utility.
+   Extract the ``access_token`` field to retrieve the necessary access token.
+   Pay attention to the ``expires_in`` field to note the number of seconds before the token expires.
+
+2. Generate MinIO Credentials using the ``AssumeRoleWithWebIdentity`` API
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      curl -X POST "https://|MINIO_S3_URL|" \
+           -H "Content-Type: application/x-www-form-urlencoded" \
+           -d "Action=AssumeRoleWithWebIdentity" \
+           -d "Version=2011-06-15" \
+           -d "DurationSeconds=86000" \
+           -d "WebIdentityToken=TOKEN"
+
+   Replace the ``TOKEN`` with the ``access_token`` value returned by Keycloak.
+
+   The API returns an XML document on success containing the following keys:
+   
+   - ``Credentials.AccessKeyId`` - the Access Key for the Keycloak User
+   - ``Credentials.SecretAccessKey`` - the Secret Key for the Keycloak User
+   - ``Credentials.SessionToken`` - the Session Token for the Keycloak User
+   - ``Credentials.Expiration`` - the Expiration Date for the generated credentials.
+
+3. Test the Credentials
+
+   Use your preferred S3-compatible SDK to connect to MinIO using the generated credentials.
+
+   For example, the following Python code using the MinIO :ref:`Python SDK <minio-python-quickstart>` connects to the MinIO deployment and returns a list of buckets:
+
+   .. code-block:: python
+      :substitutions:
+
+      from minio import Minio
+
+      client = MinIO(
+         "|MINIO_S3_URL|",
+         access_key = "ACCESS_KEY",
+         secret_key = "SECRET_KEY",
+         session_token = "SESSION_TOKEN"
+         secure = True
+      )
+
+      client.list_buckets()
+
+.. end-configure-keycloak-sts
+
+.. start-configure-keycloak-minio-console
+
+Log in as a user with administrative privileges for the MinIO deployment such as a user with the :userpolicy:`consoleAdmin` policy.
+
+Select :guilabel:`Identity` from the left-hand navigation bar, then select :guilabel:`OpenID`.
+Select :guilabel:`Create Configuration` to create a new configuration.
+
+Enter the following information into the modal:
+
+.. list-table::
+   :stub-columns: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - :guilabel:`Name` 
+     - Enter a unique name for the Keycloak instance 
+   
+   * - :guilabel:`Config URL`
+     - Specify the address of the Keycloak OpenID configuration document (|KEYCLOAK_URL|)
+
+       Ensure the ``REALM`` matches the Keycloak realm you want to use for authenticating users to MinIO
```

Comment:
While the bullet shouldn't have a `.`, I would end the extra note here and in each of the below bullets with one. Especially since you use periods for the last point.